### ISSUE TITLE
[Accessibility- Navigation 12.6]Breadcrumb cannot be reached.

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3757]
### Problem Statement

Environment: Tribe

#### Steps to Reproduce
1. Create a tree of notes
2. Access the last note in the tree

#### Current behavior
The breadcrumb in the top of page, can't be reached

#### Expected behavior

Add a role="navigation" and an aria-label="Breadcrumbs" to the div parent  of the breadcrumbs
